CharacterCreation/04_Backgrounds.txt

```diff
@@ -67,12 +67,13 @@ Outfits can include both footwear and outerwear.
 +------------------------+------------------------------------------------------------------------------+
 | Tundari society is concentrated primarily on Tundar Prime, but your family set out into the galaxy.   |
 | Whether by choice or by exile, they have built a new home and you grew up surrounded by diverse alien |
-| species, carving a life out of recently-terraformed rock.						|
+| species, carving a life out of recently-terraformed rock.						                        |
 +------------------------+------------------------------------------------------------------------------+
 | Colony World- Opaleite | Gain Languages- Opaleite Trade: Vocal or Opaleite Trade: Sign (+ 1 dialect), |
 |                        | 1 Common or Trade Language of choice. Gain 0.5 set of Nanites.               |
 +------------------------+------------------------------------------------------------------------------+
-|                    |
+| The distant comet-mining colonies of the Opalite Coalition is where you hail from. You grew up either |
```

Comment:
I'd phrase this as "You hail from one of the Opaleite Coalition's distant comet-mining colonies." Active voice feels more natural here.